lib/airrecord/client.rb

```diff
@@ -32,22 +32,79 @@ def connection
       }
     end
 
+    # TODO: remove in v2
```

Comment:
These methods were all undocumented and _probably_ only ever called internally. But I guess we should wait until v2 to remove them, in case anyone has been calling `Airrecord::Client.connection.handle_error` directly.